doc/opencv.bib

```diff
@@ -1133,15 +1496,17 @@ @article{Yuen90
   journal = {Image and Vision Computing},
   volume = {8},
   number = {1},
-  publisher = {Elsevier}
+  publisher = {Elsevier},
+  url = {https://typeset.io/pdf/a-comparative-study-of-hough-transform-methods-for-circle-2mzwteha3d.pdf}
```

Comment:
returns 403

Reply:
Same, the link is ok from me:

![image](https://github.com/user-attachments/assets/172fee9b-e079-4e4a-b67d-ff54557637b6)